typeform/form_answer.py

```diff
@@ -0,0 +1,30 @@
+class FormAnswer(object):
+    """TypeForm response answer object"""
+    __slots__ = ['_question', '_question_id', '_answer']
```

Comment:
yeah, so what `__slots__` does is explicitly set which properties exist on this class, so when you create a new instance of the class instead of having a dynamically sized object, it'll create a fixed size object.